### PR TITLE
Added BiLSTM model for bias classification

### PR DESCRIPTION
**Pull request by vartika1801**
Model Details

- **Model Type**: BiLSTM with pretrained Word2Vec embeddings (Google News 300)
- **Embedding Dimension**: 300
- **Input Length**: 300 tokens
- **Output Classes**: 3 (Left, Center, Right)
- **Accuracy on Test Data**: 73%
- **Accuracy on Training Data**: 94%

Files Include:
- bias_model.h5
-  tokenizer.pkl
- label_encoder.pkl

Files placed under bias_detection

### CODE REVIEW

**Comment by vartika1801** [comment]
sure! If there's a need for further improvements, I'd be glad to continue contributing  especially enhancing the accuracy of the model. I'm also open to contributing to other areas of the project wherever help is needed.

**Comment by vartika1801** [comment]
Sure. I would be happy to contribute. Also , I realized afterward that my Git was configured with my local machine email, so the contribution didn’t get linked to my GitHub profile.I’ve fixed that now, but if possible, I’d really appreciate it if you could add my name to the contributors list manually.

**Comment by vartika1801** [comment]
just generated a PR

**Comment by IkkiOcean** [comment]
> just generated a PR

you added a submodule to project. just add a space in some page or readme. i cant approve addition of submodule

**Comment by vartika1801** [comment]
I've raised a PR with a small change in the readme to reflect my contribution. Kindly review and acknowledge it so I can be added to the contributors list.
